handlers.go

```diff
@@ -566,9 +608,9 @@ func (s Server) DeleteBlob(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var size int64
-	if s.Prometheus {
+	if s.Prometheus || s.MaxRepoSize > 0 {
 		stat, err := os.Stat(path)
-		if err != nil {
+		if err == nil {
```

Comment:
Technically unrelated to my PR's purpose, but this fixes what I think was a bug that existed before, and with my PR, would affect the size counting if left unfixed.

Reply:
Oh, great catch